### DISCUSSION TITLE
Can support washing machine haier hwd120-b1558u ?

### DISCUSSION THREAD

**Discussion by lynxchat4** [ofalvai/home-assistant-candy#113] [Device support improvements]
I have haier hwd120-b1558u. 
Default app is U+ in PlayStore.
I thins new platform is haier-iot, I find some github project
https://github.com/haier-iot

But I scan ports and find many intresting.
open ports:
23 - telnet - QNX Neutrino need to know login add password.
8000 - qconn remote IDE support  - in information name of board "Freescale i.MX6Q Sabre Board for Smart Devices"
56800 - send some information like this:
"4bb93853d9ff425f8df99e03a6eda4b698AAFC022965" where 98AAFC022965 - mac address.

And I find another projects for Haier AC where used 56800 port
https://github.com/KoalaBear84/HaierAC
https://github.com/roeij/py-haier-ac-remote

Can you say can you integration work with my washing machine??

Maybe you know some integrations that work at least with the U+ cloud service (haier-iot)?
I only need to get the status of the machine at the moment, because she is far away in the basement and everyone forgets about her.

**Comment by ofalvai** [comment]
The best way to check compatibility is to download the [Candy Simply-Fi](https://play.google.com/store/apps/details?id=it.candy.simplyfi) app and try connecting to the device. This integration uses the same local APIs as the app.

**Reply by lynxchat4** [reply]
I can't add in app because I buyd refublished washing machine where erase all serials (on body).
Maybe you know how to get serial another way?
I try to use CandySimplyFi-tool to get key, but this tool call to 80 port. It closed on my device.

**Reply by ofalvai** [reply]
In this case I don't think your device is compatible with this integration sadly.

**Reply by lynxchat4** [reply]
Thank you!
For now, I decided to control the process of completing the wash with a regular ping. Because it disconnects from the network after 10 minutes of inactivity.

But if you find out some information about Haier U +, I hope you will not forget that I was right-handed and tell me which way to go.
